compiler/rustc_trait_selection/src/solve/eval_ctxt/canonical.rs

```diff
@@ -323,7 +323,7 @@ impl<'tcx> EvalCtxt<'_, 'tcx> {
             nested_goals.extend(
                 infcx
                     .at(&ObligationCause::dummy(), param_env)
-                    .eq(DefineOpaqueTypes::No, orig, response)
```

Comment:
Ignore all of the call sites in `rustc_trait_selection/src/solve`. Those can be commented saying that the new trait solver ignores `DefineOpaqueTypes`.